lua/null-ls/builtins/diagnostics/pylama.lua

```diff
@@ -15,26 +15,32 @@ return h.make_builtin({
         check_exit_code = function(code)
             return code < 1
         end,
-        on_output = h.diagnostics.from_json({
-            attributes = {
-                message = "message",
-                row = "lnum",
-                col = "col",
-                severity = "etype",
-                code = "number",
-                source = "source",
-            },
-            severities = {
-                E = h.diagnostics.severities["error"],
-                W = h.diagnostics.severities["warning"],
-                S = h.diagnostics.severities["warning"],
-                I = h.diagnostics.severities["warning"],
-                C = h.diagnostics.severities["warning"],
-                T = h.diagnostics.severities["warning"],
-                F = h.diagnostics.severities["information"],
-                D = h.diagnostics.severities["information"],
-                R = h.diagnostics.severities["information"],
-            },
-        }),
+        on_output = function(params)
+            if params.err == "failed to decode json: Expected value but found invalid token at character 1" then
+                params.err = nil
+                params.output = { {} }
```

Comment:
```suggestion
                return {}
```
We should be able to just return an empty able here instead of passing it through the parser.

Reply:
Don't we want to return params with output = {}?

Reply:
Nope {} is correct. My bad.